packages/moonstone/VideoPlayer/VideoPlayer.js

```diff
@@ -1644,38 +1608,47 @@ const VideoPlayerBase = class extends React.Component {
 			noMiniFeedback,
 			noSlider,
 			noSpinner,
-			preloadSource,
-			source,
 			spotlightDisabled,
 			spotlightId,
 			style,
 			thumbnailComponent,
 			thumbnailSrc,
 			title,
-			videoComponent,
-			...rest} = this.props;
-
-		delete rest.announce;
-		delete rest.autoCloseTimeout;
-		delete rest.feedbackHideDelay;
-		delete rest.jumpBy;
-		delete rest.miniFeedbackHideDelay;
-		delete rest.onControlsAvailable;
-		delete rest.onFastForward;
-		delete rest.onJumpBackward;
-		delete rest.onJumpForward;
-		delete rest.onPause;
-		delete rest.onPlay;
-		delete rest.onRewind;
-		delete rest.onScrub;
-		delete rest.onSeekFailed;
-		delete rest.pauseAtEnd;
-		delete rest.playbackRateHash;
-		delete rest.seekDisabled;
-		delete rest.setApiProvider;
-		delete rest.thumbnailUnavailable;
-		delete rest.titleHideDelay;
-		delete rest.videoPath;
+			videoComponent: VideoComponent,
+			...mediaProps
+		} = this.props;
+
+		delete mediaProps.announce;
+		delete mediaProps.autoCloseTimeout;
+		delete mediaProps.children;
+		delete mediaProps.feedbackHideDelay;
+		delete mediaProps.jumpBy;
+		delete mediaProps.miniFeedbackHideDelay;
+		delete mediaProps.onControlsAvailable;
+		delete mediaProps.onFastForward;
+		delete mediaProps.onJumpBackward;
+		delete mediaProps.onJumpForward;
+		delete mediaProps.onPause;
+		delete mediaProps.onPlay;
+		delete mediaProps.onRewind;
+		delete mediaProps.onScrub;
+		delete mediaProps.onSeekFailed;
+		delete mediaProps.pauseAtEnd;
+		delete mediaProps.playbackRateHash;
+		delete mediaProps.seekDisabled;
+		delete mediaProps.setApiProvider;
+		delete mediaProps.thumbnailUnavailable;
+		delete mediaProps.titleHideDelay;
+		delete mediaProps.videoPath;
+
+		mediaProps.autoPlay = !noAutoPlay;
```

Comment:
I am not sure why, but after properly setting `noAutoPlay` prop to `Video`, it seems like loading gets "suspended" if `autoPlay=false`. Calling `load()` on mount and src update seem to resolve the problem. Where is the `suspend` event coming from??

Reply:
Good catch on the prop. I elected to keep this line and change Video to use `autoPlay` instead of `noAutoPlay`. That keeps Video consistent with Media (on which it's based).

Regarding the issue you're seeing, I'm not following what the problem is. I see the `suspend` event firing but it doesn't appear to cause problems nor necessarily indicate a problem based on the docs: https://developer.mozilla.org/en-US/docs/Web/Events/suspend.